tests/test_app/test_typing.py

```diff
@@ -1,4 +1,4 @@
-from typing import List, Tuple, Union
+from typing import List, Set, Tuple, Union
```

Comment:
Why we don't use   list, tuple, set  here?

Reply:
we do in the tests [see here](https://github.com/cogent3/cogent3/pull/1200/files#diff-f28422feb4ea07b15d9eb51efd9c0efdcd0f73ee20856cf1117e3f407403bfaaR101)

Reply:
OK. Thank you.

Reply:
now expanded those into 2 separate tests since the type subscripting doesn't work for python 3.8, using pytests skip decorator